orm/encoding/ormkv/entry.go

```diff
@@ -56,19 +61,7 @@ func fmtValues(values []protoreflect.Value) string {
 
 	parts := make([]string, len(values))
 	for i, v := range values {
-		val, err := structpb.NewValue(v.Interface())
-		if err != nil {
-			parts[i] = "ERR"
-			continue
-		}
-
-		bz, err := protojson.Marshal(val)
-		if err != nil {
-			parts[i] = "ERR"
-			continue
-		}
-
-		parts[i] = string(bz)
+		parts[i] = fmt.Sprintf("%v", v.Interface())
```

Comment:
Another slight change in the format of the entry and `entry_test.go` needs to be updated (the key no longer has double quotes around string values, e.g. `1/"abc"` becomes `1/abc`).